spec/u3d/iniparser_spec.rb

```diff
@@ -82,5 +82,36 @@
         end
       end
     end
+
+    describe '.create_linux_ini', focuss: true do
```

Comment:
Remove focus tag if this is not meant to stay?

Reply:
Sure. I forgot it.